### ISSUE TITLE
Can we manually map Hypothalamus to the correct MNI space instead of the thalamus?

### ISSUE THREAD

**Issue by thenineteen** [thenineteen/Semiology-Visualisation-Tool#186]
As the GIF parecelllations we've used don't have a label for the hypothalamus, our current default mapping (#75) is to show the thalami. 

Is there a way to manually highlight the region of interest? e.g. we create a new GIF parecellation number 999 and map this to the region bounded by the approximate points below:

MNI_152 = (94, 134, 63), (94, 134, 70), (94, 127, 63) and (94, 126, 70)

**Comment by fepegar** [comment]
Does Freesurfer segment the hypothalamus?

**Comment by thenineteen** [comment]
> Does Freesurfer segment the hypothalamus?

not sure, tried looking for it in various places...

**Comment by thenineteen** [comment]
... someone in the comments here seems to suggest they have an MNI mask for the hypothalamus - would that help?

**Comment by fepegar** [comment]
Yes. But we would need to adapt the GIF parcellation and the colour table.

**Comment by thenineteen** [comment]
> Yes. But we would need to adapt the GIF parcellation and the colour table.

That would be ideal if possible

**Comment by fepegar** [comment]
Then we need a segmentation in MNI space.

**Comment by thenineteen** [comment]
> ... someone in the comments here seems to suggest they have an MNI mask for the hypothalamus - would that help?
> 
> https://www.researchgate.net/post/Does_anyone_know_of_an_available_mask_of_the_hypothalamus_If_not_how_do_people_recommend_creating_one

does the above help?

![image](https://user-images.githubusercontent.com/10786099/91532817-c73d9980-e906-11ea-9132-6de8942694fc.png)

**Comment by fepegar** [comment]
Maybe. We don't know how they generated that.
